patch/tree-sitter-0.22.6/Makefile.patch

```diff
@@ -0,0 +1,53 @@
+--- ../../downloads/tree-sitter-0.22.6/Makefile.orig	2024-05-05 13:47:18.000000000 -0700
++++ ../../downloads/tree-sitter-0.22.6/Makefile	2024-06-18 19:52:35.772310737 -0700
+@@ -8,12 +8,12 @@
+
+ # collect sources
+ ifneq ($(AMALGAMATED),1)
+-	SRC := $(wildcard lib/src/*.c)
```

Comment:
looks like differences in spaces; would have been better to keep the diff to the minimum, but whatever.

Reply:
ok, I saw your PR on tree-sitter; this is a fix for windows. thx.